resources/views/components/quantity/enhanced.blade.php

```diff
@@ -6,27 +6,31 @@
 ])
 
 <quantity-select :input-ref="'qty-select-' + {{ $index }}" :model="parseInt({{ $model }})" :min-qty="{{ $minSaleQty }}" :increment="{{ $qtyIncrements }}" v-slot="qtySelect">
-    <div class="flex">
-        <x-rapidez::button.outline @click.prevent="qtySelect.decrease">
-            <div class="bg-neutral h-0.5 w-3 hover:bg-white"></div>
-        </x-rapidez::button.outline>
+    <div class="flex items-center justify-center border rounded bg-white h-14 self-start">
+        <button
+            @click.prevent="qtySelect.decrease"
+            v-bind:disabled="qtySelect.model <= qtySelect.defaultQty"
+            class="shrink-0 pl-2.5 text-neutral disabled:cursor-not-allowed disabled:text-neutral/50"
+        >
+            <x-heroicon-o-minus-small class="mt-0.5 size-5" stroke-width="2" />
+        </button>
         <x-rapidez::input
             name="qty"
```

Comment:
isn't it easier to just use an <input instead of the component since it's a really base version without any styling just wondering?